src/FeaturePlugin.php

```diff
@@ -54,6 +54,15 @@ protected function __construct() {}
 	 * Init the feature plugin, only if we can detect both Gutenberg and WooCommerce.
 	 */
 	public function init() {
+		/**
+		 * Filter allowing WooCommerce Admin to be disabled.
+		 *
+		 * @param bool $disabled False.
+		 */
+		if ( apply_filters( 'woocommerce_analytics_disabled', false ) ) {
```

Comment:
@rrennick If this filter is to disable all of WooCommerce Admin (and not just the analytics feature), shouldn't this filter be something like `woocommerce_admin_disabled`?

Reply:
@justinshreve I named it consistent with #3339 which hasn't been merged yet.

Reply:
@justinshreve Gotcha, thanks for catching that. I created #3361 to rename the filter.

Reply:
@rrennick I missed this PR during the review phase, but will this still work in the context when the feature is pulled in via Packagist in Woo core?

Reply:
> will this still work in the context when the feature is pulled in via Packagist in Woo core?

Yes, it should.